web-api/src/business/useCases/user/queueEmailUpdateAssociatedCasesWorker.ts

```diff
@@ -0,0 +1,111 @@
+import { AuthUser } from '@shared/business/entities/authUser/AuthUser';
+import {
+  Practitioner,
+  RawPractitioner,
+} from '@shared/business/entities/Practitioner';
+import { ROLES } from '@shared/business/entities/EntityConstants';
+import { RawUser, User } from '@shared/business/entities/User';
+import { ServerApplicationContext } from '@web-api/applicationContext';
+
+async function disableIsUserUpdatingFlag({
+  applicationContext,
+  user,
+}: {
+  applicationContext: ServerApplicationContext;
+  user: RawUser | RawPractitioner;
+}): Promise<void> {
+  user.isUpdatingInformation = false;
+  let userEntity;
+  if (
+    user.role === ROLES.privatePractitioner ||
+    user.role === ROLES.irsPractitioner ||
+    user.role === ROLES.inactivePractitioner
```

Comment:
This isn't really related to this bugfix, but I've seen this check in a few places, and I feel nervous about continuing this pattern. (It is very easy for a dev to think, "Ah, let me create a User" without realizing they need to create a _Practitioner_ instead, or it is easy for a dev to forget one of the disjuncts like `ROLES.inactivePractitioner`.)

My first thought is that it would be nicer if we could construct the appropriate user with a factory, but that's more complicated and outside your ticket's scope. However, I wonder about your thoughts pulling this check (here and wherever else it is used, which includes admin.ts, changePasswordInteractor, and updateAssociatedCaseWorker) into a function like "isPractitionerUser" or something, which is at least a little cleaner.